infrastructure/main.tf

```diff
@@ -126,3 +126,26 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
   value        = "${module.reform-blob-router-db.postgresql_database}"
 }
 # endregion
+
+# region SMTP secrets
+
+data "azurerm_key_vault_secret" "smtp_username" {
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  name         = "reports-email-username"
+}
+
+data "azurerm_key_vault_secret" "smtp_password" {
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  name         = "reports-email-password"
+}
+
+# endregion
+
+# region reports  secrets
+
+data "azurerm_key_vault_secret" "reports_recipients" {
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  name         = "reports-recipients"
+}
```

Comment:
It looks like passing these secrets to the application is missing.

Before merging this PR, please make sure these secrets are set in vaults for aat, prod, demo and perftest.

Reply:
these resources will not be used as part of terraform setup - this file change is completely not needed. secrets need to be set manually to the vault and charts, i believe, already requesting them to be present (i recall there being a separate pr)